packages/react-components/src/components/styles/DomainPermissions.styles.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IIconStyles, ILinkStyles, IStackStyles, ITextStyles } from '@fluentui/react';
+import { _pxToRem } from '@internal/acs-ui-common';
+
+/**
+ * @internal
+ */
+export const iconContainerStyles: IStackStyles = {
+  root: {
+    margin: 'auto',
+    position: 'relative'
+  }
+};
+
+/**
+ * @internal
+ */
+export const textContainerStyles: IStackStyles = {
+  root: {
+    margin: 'auto',
+    textAlign: 'center'
+  }
+};
+
+/**
+ * @internal
+ */
+export const iconPrimaryStyles: IIconStyles = {
+  root: {
+    position: 'absolute',
+    color: '#2B88D8',
```

Comment:
use theme here

Reply:
Fixed in the last PR introducing the sparkle icon: https://github.com/Azure/communication-ui-library/pull/2307/files